apps/user-profile/src/components/common/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuItem,
+} from '@devlaunchers/components/components/DropdownMenu';
+function DropDownMenu({ menuItems = [], country, onCountryChange }) {
```

Comment:
`({ menuItems = [], **country**, **onCountryChange** })` those last two parameters should not be coupled to the "country" solution.  Remember this is a component that will be used throughout the the user profile project for different tasks.